scss/pixel-arts/coin.scss

```diff
@@ -0,0 +1,41 @@
+.nes-coin {
+  $coin-colors: (#060606,#ffd700, #daa520);
+  // prettier-ignore
+  $coin: (
+    (0,0,0,0,0,1,1,1,1,0,0,0,0,0),
+    (0,0,0,1,1,2,2,2,2,1,1,0,0,0),
+    (0,0,1,2,2,1,1,1,1,2,2,1,0,0),
+    (0,1,2,1,1,3,3,3,3,1,2,1,1,0),
+    (0,1,2,1,3,3,3,3,3,3,1,2,1,0),
+    (1,2,1,3,3,3,3,3,3,3,3,1,2,1),
+    (1,2,1,3,3,3,3,3,3,3,3,1,2,1),
+    (1,2,1,3,3,3,3,3,3,3,3,1,2,1),
+    (1,2,1,3,3,3,3,3,3,3,3,1,2,1),
+    (0,1,2,1,3,3,3,3,3,3,1,2,1,0),
+    (0,1,2,1,1,3,3,3,3,1,2,2,1,0),
+    (0,0,1,2,2,1,1,1,1,2,2,1,0,0),
+    (0,0,0,1,1,2,2,2,2,1,1,0,0,0),
+    (0,0,0,0,0,1,1,1,1,0,0,0,0,0),
+  );
```

Comment:
Could you make this icon at 16x16? 🤔 
and Please use `.nes-icon .coin` instead of `.nes-coin`. 🙏 

Other people may also want to use your coin icon 👍

Reply:
Sorry for my feeling opinion. (not logical)
It's difficult to understand for me that it is a coin. 🤔 
![2019-02-09 23 16 40](https://user-images.githubusercontent.com/5305599/52521910-fc9d7a80-2cc1-11e9-9c29-426e7372e1ed.png)

@guastallaigor Do you have any good ideas? 


https://www.google.com/search?q=pixel+art+coin&gl=us&hl=en&pws=0&source=lnms&tbm=isch&sa=X&ved=0ahUKEwjUl--w767gAhXTdN4KHYuwDRQQ_AUIDigB&biw=960&bih=1066

Reply:
I'm not sure. 8bit coin I only know the Mario one :smile:.
But tomorrow I can look it up something, the idea here is good thought, i like it :+1:

Reply:
Oh, srry about the file name // class name.
I copy/pasted the `pokeball`, run a regex to change *pokeball* to *coin*, and changed the colors ^^

Reply:
8-bit Nintendo style coin for the win!
(If you have a tool for generating those color maps, can you pont me to it?)

Reply:
I would still like the `.nes-icon`s to have some spinning animation.

This is what I would use if accepted:
![spinning](https://user-images.githubusercontent.com/18617120/52526742-8eef4f80-2cbd-11e9-8708-2dbfc7b896f2.gif)
 
I already have a local commit that adds this :) 
Say the word, and I push it.

Otherwise, good to merge! :rocket:

Reply:
> If you have a tool for generating those color maps, can you point me to it?

There is no tool that can output in matrix like this project. 😭 
But there is a tool made by @guastallaigor .
https://vue-pixel-art.now.sh/

Reply:
_resolving as it is done_